docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,391 @@
+  6. In the App location dropdown, select the database and schema for your app.
+  7. Select Create. The Streamlit in Snowflake editor opens an example Streamlit application in Viewer mode. Viewer mode allows you to see how the Streamlit application appears to users.
+  8. Click Edit to create your application.
+
+##### Step 2: Install required packages:
+  [PUT](https://docs.snowflake.com/en/sql-reference/sql/put) the [environment.yml](/environment.yml) file to the stage location specified by the `STREAMLIT_ROOT_LOCATION` parameter of the Streamlit object:
+  ```
+  $ SELECT STREAMLIT_ROOT_LOCATION FROM <database>.INFORMATION_SCHEMA.STREAMLITS;
+
+  > <database>.<schema>."<appname> (Stage)"
+  ```
+  ```
+  PUT file:///<full_path>/environment.yml '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+##### Step 4: Provide the geojson [file](/countries.geo.json) that contains the country information required to render the choropleth map:
+  ```
+  PUT file:///<full_path>/countries.geo.json '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+##### Step 5: Copy the code from [snowplow.py](/snowplow.py) inside the editor
+
+and Click `Run`
+
+</details>
+
+<details>
+
+<summary>Option 3. Run it on Streamlit's cloud</summary>
+
+Follow the instruction on [https://streamlit.io/cloud](https://streamlit.io/cloud)
+
+</details>
+
+### Applying filters
+
+#### Time filters
+
+By default the app is configured to always retrieve the last 7 days data. Users are also able to modify the start and end date of this filter to modify this range at any point in time, which will rerun all queries and applies the filters to all visualisations.
+
+## App_id filters
+
+Users could also filter on app_ids. To change the default two items, replace appid1 and appid2:
+
+    app_id1 = st.text_input("App ID:", key = 'appid1')
+    app_id2 = st.text_input("App ID:", key = 'appid2')
+
+
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+</TabItem>
+
+</Tabs>
+
+## Dashboards
+
+### Acquisition Overview
+
+The **Acquisition Overview** provides an overview on the website traffic, with a breakdown by marketing channels.
+
+There is also a distinction between `New Users` who have not previously had a session on the website (`domain_sessionid = 1`) and `Existing users` who have a session with `domain_sessionidx > 1`.
+
+<Tabs groupId="dashboards" queryString>
+
+<TabItem value="PowerBI" label="PowerBI">
+
+![](images/PBI_AO.png)
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+![](images/T_AO.png)
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+![](images/S_AO.png)
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+![](images/P_AO.png)
+
+</TabItem>
+
+</Tabs>
+
+### User Acquisition
+This view is about **User Acquisition** which is effectively about what channels / methods were used to acquire new users.
+
```

Comment:
We're a little inconsistent in the first line of each section, we don't name the type above, here we use a view, below we use a page. I know they're the same thing but we should probably pick one and stick with it